src/components/inputComponents/inputRecipe.js

```diff
@@ -0,0 +1,11 @@
+export default function InputRecipe() {
+  return (
+    <textarea
+      id="inputRecipe"
+      name="inputRecipe"
```

Comment:
`cakeRecipe`
As you can see, i'm just trying to be consistent

Reply:
Yes, I see your point. The suggested naming is much clearer.